BookPlayer/Services/BookSortService+SortError+PlayListSortOrder.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+final class BookSortService {
+
+    var books: NSOrderedSet
```

Comment:
I'm still debating if this class needs a property `books` 🤔, since an instance of this class is actually only used immediately for the `perform` function and then never used again. Wouldn't it make more sense to make `perform` a class function, and pass the books as a parameter? thoughts?

Reply:
I agree. One of those situations where you start the class and put what you think you need in place and end up not needing it in the end.